controllers/test_variables.go

```diff
@@ -36,17 +36,19 @@ var (
 	ibmObjectCSICRName    = "test-csi-cr"
 	ibmObjectCSIfinalizer = "ibmobjectcsi.objectdriver.csi.ibm.com"
 
-	recoverStaleVolCRName   = "test-vol-cr"
-	testDeploymentName      = "test-deployment"
-	testDeploymentNamespace = "default"
-	testPVName1             = "test-pv-1"
-	testPVName2             = "test-pv-2"
-	testPVCName1            = "test-pvc-1"
-	testPVCName2            = "test-pvc-2"
-	testStorageClassName    = "test-csi-storage-class"
-	testNode1               = "test-node-1"
-	testNode2               = "test-node-2"
-	testNode3               = "test-node-3"
+	/*
+		recoverStaleVolCRName   = "test-vol-cr"
+		testDeploymentName      = "test-deployment"
+		testDeploymentNamespace = "default"
+		testPVName1             = "test-pv-1"
+		testPVName2             = "test-pv-2"
+		testPVCName1            = "test-pvc-1"
+		testPVCName2            = "test-pvc-2"
+		testStorageClassName    = "test-csi-storage-class"
+		testNode1               = "test-node-1"
+		testNode2               = "test-node-2"
+		testNode3               = "test-node-3"
+	*/
 )
```

Comment:
can we remove this if not used in code @ashimagarg27  ?

Reply:
Actually these we need for UTs, I have not fixed UTs in this PR because once the code changes are approved I will deliver UTs in separate PR